project.mml

```diff
@@ -1005,6 +1006,45 @@ Layer:
         ) AS guideways
     properties:
       minzoom: 11
+  - id: roller-coaster-casing
+    geometry: linestring
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+            way,
+            COALESCE(layer,0) AS layernotnull,
+            tags->'roller_coaster' AS roller_coaster,
+            railway,
+            CASE WHEN (tunnel = 'yes' OR tunnel = 'building_passage' OR covered = 'yes') THEN 'yes' ELSE 'no' END AS tunnel,
+            CASE WHEN (bridge = 'yes' OR bridge = 'covered' OR bridge = 'viaduct') THEN 'yes' ELSE 'no' END AS bridge
+          FROM planet_osm_line
+          WHERE railway IS NULL
```

Comment:
The WHERE clause must restrict this layer to only roller coasters.

I would also prefer not to see the railway IS NULL check, as tagging a rollercoaster as a railway is wrong generally.